### PR TITLE
add about us

### PR DESCRIPTION
**Pull request by mazurkevych30**

### CODE REVIEW

File: src/css/layout/about_us.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    margin-bottom: 20px;
+}
+
+.about-us-item-text {
+    margin-top: 16px;
+    color: var(--black);
+    font-size: 20px;
+    font-weight: 700;
+    line-height: 1.33;
+    letter-spacing: -0.36px;
```
Comment (Iryna-Holova) [inline src/css/layout/about_us.css]: use em

File: src/css/layout/about_us.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+.about-us-item::before {
+    content: counter(item, decimal-leading-zero);
+    counter-increment: item;
+    color: var(--green);
+    font-weight: 700;
+    font-size: 28px;
+    font-weight: 700;
+    line-height: 1.14;
+    letter-spacing: -0.56px;
```
Comment (Iryna-Holova) [inline src/css/layout/about_us.css]: em

File: src/css/layout/about_us.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    content: "";
+    display: block;
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    z-index: -1;
+    width: 39px;
+    height: 12px;
+    border-radius: 8px;
+    background-color: var(--green-middle);
```
Comment (Iryna-Holova) [inline src/css/layout/about_us.css]: use common styles .section-title-accent

File: src/css/layout/about_us.css [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    letter-spacing: -0.56px;
+}
+
+.title-about-us {
+    text-align: center;
+}
+
+.title-about-us-block {
+    position: relative;
+}
```
Comment (Iryna-Holova) [inline src/css/layout/about_us.css]: use common styles .section-title-accent

File: src/partials/about_us.html [hunk 5]
```diff
@@ -0,0 +1,30 @@
+<section class="about-us">
+  <div class="container">
+    <h2 class="title-about-us section-title">
+      About <span class="title-about-us-block">Us</span>
```
Comment (Iryna-Holova) [inline src/partials/about_us.html]: class='section-title-accent'

File: src/partials/about_us.html [hunk 6]
```diff
@@ -0,0 +1,30 @@
+<section class="about-us">
+  <div class="container">
+    <h2 class="title-about-us section-title">
```
Comment (Iryna-Holova) [inline src/partials/about_us.html]: remove title-about-us

**Review by Iryna-Holova — APPROVED** [review]

**Review by Maxus94 — APPROVED** [review]

**Pull request merged**